cloudflare/schema_cloudflare_custom_hostname.go

```diff
@@ -22,6 +22,10 @@ func resourceCloudflareCustomHostnameSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Optional: true,
 		},
+		"custom_origin_sni": {
+			Type:     schema.TypeString,
+			Computed: true,
```

Comment:
why `Computed`? isn't this `Optional`?

Reply:
Thanks for the comment!
Oops, according to the [API docs](https://api.cloudflare.com/#custom-hostname-for-a-zone-edit-custom-hostname), it is so I'll change it.